app/components/events/index.hbs

```diff
@@ -0,0 +1,26 @@
+<Header />
+<main>
```

Comment:
NIT: main tag is not really required to be explicitly defined in an ember application